paf-mvp-operator-express/src/operator-node.ts

```diff
@@ -286,51 +287,58 @@ export class OperatorNode implements Node {
     };
 
     app.expressApp.get(jsonOperatorEndpoints.read, cors(corsOptionsAcceptAll), async (req, res) => {
-      logger.Info(jsonOperatorEndpoints.read);
-
-      // Attempt to set a cookie (as 3PC), will be useful later if this call fails to get Prebid cookie values
-      setTest3pcCookie(res);
-
-      const request = getPafDataFromQueryString<GetIdsPrefsRequest>(req);
-
-      try {
-        const response = await getReadResponse(request, req);
-        res.json(response);
-      } catch (e) {
-        logger.Error(jsonOperatorEndpoints.read, e);
-        // FIXME finer error return
-        const error: OperatorError = {
-          type: OperatorErrorType.UNKNOWN_ERROR,
-          details: '',
-        };
-        res.status(400);
-        res.json(error);
-      }
+      tracer.startActiveSpan('jsonOperatorEndpoints.read', async (span) => {
```

Comment:
note: already discussed offline with @KBM-Criteo , that we will encapsulate this into a method called for each endpoint